Cargo.lock

```diff
@@ -4666,17 +4704,18 @@ dependencies = [
  "polars-ops",
  "polars-plan",
  "polars-time",
+ "polars-utils",
  "rand",
  "serde",
  "serde_json",
- "sqlparser",
+ "sqlparser 0.49.0",
```

Comment:
I was thinking it would make sense to keep our `sqlparser` dependency on the same version as `polars-sql`?